### DISCUSSION TITLE
Welcome to PastVu Discussions!

### DISCUSSION THREAD

**Discussion by klimashkin** [PastVu/pastvu#328] [General / Основное]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paul-k-pastvu** [comment]
Правильное дело. Наверное, что-то из того, что сейчас висит в задачах, можно вынести в дискуссии, в категорию "Ideas". 
То, что вызывает концептуальные вопросы или чётко не сформулировано (а только в виде пожеланий на далёкое будущее).

**Reply by dinich1** [reply]
Здравствуйте, а с кем можно связаться по поводу разрешения на использование снимков с сайта PastVu?

**Reply by aeifn** [reply]
> Здравствуйте, а с кем можно связаться по поводу разрешения на использование снимков с сайта PastVu?

Пишите на support@pastvu.com

**Reply by paul-k-pastvu** [reply]
Сайт PastVu - в "серой зоне". На сайте есть фото из семейных архивов, есть сканы различных публикаций, есть фото, скопированные с других сайтов. Нужно смотреть по каждому фото, что там указано в поле "источник", и, возможно, обращаться к тому участнику, который это фото загрузил.

Правила сайта [https://pastvu.com/rules](https://pastvu.com/rules) гласят следующее:

> 1.15. Пользователь, размещая свои материалы на сайте, безвозмездно разрешает их дальнейшее использование пользователями сайта в личных целях, не связанное с извлечением коммерческой выгоды: воспроизводство в память ЭВМ, в память системы мобильной сотовой связи, иное воспроизведение в личных целях.
> 1.16. Любое использование опубликованных пользователем материалов в коммерческих целях возможно только на основании соответствующего разрешения правообладателя конкретного изображения. 
> 1.20. Любой пользователь может просматривать и сохранять у себя на компьютере (копировать к себе на компьютер) опубликованные изображения и другие материалы. Администрация ни при каких условиях не несет ответственности за дальнейшее использование материалов, полученных с сайта.

В остальном - на свой страх и риск. Администрация сайта ни в коем случае не является ни агентом, ни посредником в вопросах использования изображений.

**Comment by n1kub** [comment]
Hey, I'm trying to dive deeper into your project project architecture, but can't find any discription. I tried to search in docs and had no success. Reading the source also was a bad idea, because I'm not javascript developer. So the question is where can I find detailed project description?

**Reply by aeifn** [reply]
Hello! It depends on what do you want to do. For a global architecture you may inspect docker-compose.yaml file and run the project locally.

**Reply by n1kub** [reply]
@aeifn I'll have a look. Thanks

**Reply by kabalin** [reply]
Some architecture related info is also here https://docs.pastvu.com/dev/setup.
